crates/fuel-core/src/graphql_api.rs

```diff
@@ -59,7 +59,7 @@ pub struct ServiceConfig {
     pub costs: Costs,
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
```

Comment:
Do we want copy on something this big? This can be over 1280 bits.

Reply:
160 bytes sounds like something small and shouldn't affect eprformance=)

Reply:
If you say so. I don't have any rules for how big something should be to only implement `Clone`, but this is just bigger than most things I see.